models/issues/milestone.go

```diff
@@ -326,6 +380,15 @@ func DeleteMilestoneByRepoID(repoID, id int64) error {
 	if _, err = db.Exec(ctx, "UPDATE `issue` SET milestone_id = 0 WHERE milestone_id = ?", m.ID); err != nil {
 		return err
 	}
+
+	if err = m.LoadLabels(); err != nil {
```

Comment:
Need `ctx`

Reply:
Any idea why build failing? It says not enough arguments to db.TxContext(), but there are no arguments to be passed to db.TxContext()

Reply:
@lunny have you seen my question? I tried to make all calls with context but same error was produced